### PR TITLE
add new term clonal hematopoiesis

### LINKED ISSUE #7137: Request for new term [clonal hematopoiesis]
### Label

clonal hematopoiesis

### Synonyms

clonal hematopoiesis

### Synonym type

exact

### Definition

Cellular proliferation of hematopoietic cells where a substantial proportion of the cells is derived from a single hematopoietic stem cell lineage.
http://purl.obolibrary.org/obo/NCIT_C162188
https://pubmed.ncbi.nlm.nih.gov/34215849/

### Parent term

MONDO:0060782 premalignant hematological system disease

### Children term(s)

_No response_

### ORCID Identifier

_No response_

### Website URL

_No response_

### Additional comments

Ideally there would be a child term, clonal hematopoiesis of indeterminate potential, to match the NCIT ontology
https://www.ebi.ac.uk/ols4/ontologies/ncit/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FNCIT_C158547?lang=en

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
and children
close #7137

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -513752,6 +513752,31 @@ relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7099" xsd:anyURI
 
+[Term]
+id: MONDO:0100535
+name: clonal hematopoiesis
+def: "Cellular proliferation of hematopoietic cells where a substantial proportion of the cells is derived from a single hematopoietic stem cell lineage." [NCIT:C162188, PMID:34215849]
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:513758]: to make it clear that we are talking about the condition, should we start the definition with 
"A precancerous condition characterized by ..."

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -513752,6 +513752,31 @@ relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql
...
+xref: NCIT:C162188 {source="MONDO:equivalentTo"}
+is_a: MONDO:0060782 {source="NCIT:C162188"} ! premalignant hematological system disease
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7137" xsd:anyURI
+
+[Term]
+id: MONDO:0100536
+name: clonal hematopoiesis of indeterminate potential
+def: "A term that refers to the presence of somatic mutations in bone marrow or peripheral blood cells in individuals who may be cytopenic but do not have morphologic evidence of hematologic neoplasia. Its prevalence rises with age and is found in approximately 10% of individuals aged 70 to 80. It is associated with an increased risk of hematologic neoplasia. Mutations in the DNMT3A, TET2, or ASXL1 genes are usually identified. Approximately 10%-40% of individuals with age-related clonal hematopoiesis will progress to meet the diagnostic criteria for clonal hematopoiesis of indeterminate potential." [NCIT:C158547]
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:513767]: Make sure that the term refers to a disease. Maybe start the definition with
"A precancerous condition characterized by the presence of...."

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -513752,6 +513752,31 @@ relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql
...
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7137" xsd:anyURI
+
+[Term]
+id: MONDO:0100537
+name: age-related clonal hematopoiesis
+def: "The gradual, clonal expansion of hematopoietic stem and progenitor cells carrying specific, disruptive, and recurrent genetic variants, in individuals without clear diagnosis of hematological malignancies. It is associated with an increased risk of developing hematologic cancers." [NCIT:C155821]
+synonym: "ARCH" EXACT ABBREVIATION [NCIT:C155821]
+is_a: MONDO:0100535 {source="NCIT:C155821"} ! clonal hematopoiesis
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
+
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:513779]: need the GitHub issue tracker

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -513752,6 +513752,31 @@ relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql
...
+is_a: MONDO:0100535 {source="NCIT:C158547"} ! clonal hematopoiesis
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7137" xsd:anyURI
+
+[Term]
+id: MONDO:0100537
+name: age-related clonal hematopoiesis
+def: "The gradual, clonal expansion of hematopoietic stem and progenitor cells carrying specific, disruptive, and recurrent genetic variants, in individuals without clear diagnosis of hematological malignancies. It is associated with an increased risk of developing hematologic cancers." [NCIT:C155821]
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:513775]: Make sure that the term refers to a disease. Maybe start the definition with
"A precancerous condition characterized by ..."

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@nicolevasilevsky 
I think we need to make sure the definitions express that the terms refer to diseases (otherwise the term seem to refer to a process)

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:513758]: good suggestion, fixed

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:513767]: fixed

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:513775]: fixed

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:513779]: added

**Comment by nicolevasilevsky** [comment]
ugh, the conflicts screwed this up. I will look into this later.

**Comment by nicolevasilevsky** [comment]
ready for re-review

**Comment by nicolevasilevsky** [comment]
this is really screwed up, I'll probably need to redo this.

@sabrinatoro this is a case where this new term request sat for over a week before being reviewed and there were conflicts and now a bunch of issues due to the conflicts. Can we implement a system where new term PRs are reviewed more quickly because this is adding a lot of additional time to addressing this ticket, which should have been straightforward.

**Comment by twhetzel** [comment]
- who is supposed to be reviewing this ticket now
- there are qc errors

**Comment by nicolevasilevsky** [comment]
@twhetzel it is assigned to me, I will take care of it. I'll probably close this and start over but I don't have time to deal with it today.

**Pull request closed without merging**